libs/galley-types/src/Galley/Types/Teams.hs

```diff
@@ -297,15 +298,13 @@ makeLenses ''FeatureFlags
 -- | See Note [hidden team roles]
 data HiddenPerm
   = ChangeLegalHoldTeamSettings
-  | ViewLegalHoldTeamSettings
   | ChangeLegalHoldUserSettings
   | ViewLegalHoldUserSettings
-  | ViewSSOTeamSettings -- (change is only allowed via customer support backoffice)
-  | ViewTeamSearchVisibilityAvailable
+  | ViewTeamFeature TeamFeatureName
   | ChangeTeamSearchVisibility
   | ViewTeamSearchVisibility
```

Comment:
```suggestion
```

Reply:
`ViewTeamSearchVisibility` and `ViewTeamSearchVisibilityAvailable` mean different things for some reason. Only `ViewTeamSearchVisibilityAvailable` corresponds to the flag itself, which is the scope of this PR.

Reply:
I think it's correct that he only removed `ViewTeamSearchVisibilityAvailable`, as this is the team-level feature that can only be enabled via the backoffice (like the other feature flags).
`ViewTeamSearchVisibility` can be enabled by admins.